docs/built-in-models/linear.md

```diff
@@ -28,6 +28,9 @@ The `n_bits` parameter controls the bit-width of the inputs and weights of the l
 
 The `n_bits` parameter can be set to `8` or more bits for models with up to `300` input dimensions. When the input has more dimensions, `n_bits` must be reduced to `6-7`. All performance metrics are preserved down to `n_bits=6`, compared to the non-quantized float models from scikit-learn.
 
+The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to make all feature distribution similar by using standard or min-max normalization.
+For a more detailed comparison of the impact of such pre-processing please refer to [the logistic regression notebook](../advanced_examples/LogisticRegression.ipynb).
```

Comment:
could be worth to include the graphs from the notebook here for better understanding !

Reply:
They might take too much place in the readme, what do you think about it @andrei-stoian-zama ?

Reply:
yeah indeed, well as you guys wish